### ISSUE TITLE
E1/T1 unable to maintain temperature with print cooling fan at 100%

### ISSUE THREAD

**Issue by orias** [alephobjects/print_profiles#9]
Profile used:  PP_stainless_steel_medium_0.5TAZ.ini

When I turned the fan off through the GLCD, it was able to maintain E1 hot end temp.

TAZ 5, Javelin v2 firmware flashed through latest Cura fimware flash menu.

**Comment by nickthetait** [comment]
Yeah, I'd seen this too. I was just trying to get a better test case that shows what is wrong...

**Comment by brentmac** [comment]
Looking into this. I'll see if we can bump up the heater power in firmware, otherwise lowering the fan power in the profiles might be our best bet

**Comment by brentmac** [comment]
can one of you guys test this firmware and let me know if it resolved the issue? I don't have a v2 dual at the moment to try it out on

/home/aleph/shared-j/devel/lulzbot/TAZ/accessories/javelin/software/test_firmware/Marlin_2015Q3_TAZ5_dual_heater_test

**Comment by nickthetait** [comment]
Now when you initially connect to the printer from cura, the firmware does not get restarted which is quite nice! Previously if you preheated, then connected via cura you would have to click preheat again.

**Comment by nickthetait** [comment]
Actually, nevermind. It still behaves the same as before. Connecting from cura restarts the printer :(

**Comment by Karrad** [comment]
Brent,

I will give this a go this morning. One thing I noticed the other day, 
is I was unable to draw enough power to heat the V1 dual using a Peng 
Chu PSU. We may want to test this with both versions of duals, and PSU's.

~Brent.i

Brent I.
Aleph Objects, Inc.
+1-970-377-1111 x671
support@lulzbot.com
http://Forum.LulzBot.com
http://LulzBot.com
brent.i@alephobjects.com

**Comment by Karrad** [comment]
Initial testing shows Peng Chu PSU cannot maintain temp. 295 for both heads, both fans 100%. T0 dropped to ~175 before maintaining temp. Testing with new PSU to see if there is a difference.

**Comment by Karrad** [comment]
New PSU will maintain a little better temp. Both heads 295, fans 100%  T0 will drop to ~180

**Comment by Karrad** [comment]
Nick were you seeing this on E1 as well? Also what printers are you using? CD i think you have a 4 upgraded to a 5?

**Comment by orias** [comment]
TAZ 4, upgraded to a 5, with Javelin, and latest Dually v2 fw.

**Comment by nickthetait** [comment]
I was testing a Taz 5 with Longfin
